islandora.module

```diff
@@ -2170,3 +2170,31 @@ function islandora_islandora_get_breadcrumb_query_predicates(AbstractObject $obj
     'fedora-rels-ext:isMemberOf',
   );
 }
+
+/**
+ * Implements hook_islandora_overview_object().
+ */
+function islandora_islandora_overview_object(AbstractObject $object) {
+  $cmodels = $object->models;
+  $rows = array();
+  foreach ($cmodels as $cmodel) {
+    if ($cmodel != 'fedora-system:FedoraObject-3.0') {
+      $o = islandora_object_load($cmodel);
+      if (is_object($o)) {
+        $rows[] = array(l($o->label, "islandora/object/{$o->id}"));
+      }
+      else {
+        $rows[] = array(t("@cmodel  - (This content model is not in this Islandora repository.)", array('@cmodel' => $cmodel)));
+      }
+    }
+  }
+  $header = count($rows) > 1 ? t('Content Models') : t('Content Model');
+  $table = array(
+    '#theme' => 'table',
+    '#header' => array($header),
+    '#rows' => $rows,
+    '#empty' => t('No Islandora content model associated with this object.'),
+  );
+  $rendered_table = drupal_render($table);
```

Comment:
Yeah, looks like I misunderstood... Should probably throw together a ticket for the `islandora_default_overview` bit not existing (looks like [copypasta from quite a while ago](https://github.com/Islandora/islandora/pull/267/files); even though it will _almost_ never be called, since `islandora_basic_collection` will usually be installed... It looks like it would throw warnings in the watchdog logs).

By "collections", you mean [`islandora_basic_collection`](https://github.com/Islandora/islandora_solution_pack_collection), yeah?... I may have to look into it somewhat.... Though I suspect the [`islandora_as_renderable_array()`](https://github.com/Islandora/islandora/blob/7.x/includes/utilities.inc#L822-L861)... It's a little naive assuming either `#type` or `#markup` will be present... I suspect if you add `'#type' => 'markup'` to the renderable array here, that passing through the renderable array will work.